tests/test_util_common.py

```diff
@@ -76,10 +76,22 @@ def test_format_btc_amount():
 
 def test_format_btc_amount_as_sats():
     btc_amount = 0.00560000
-    assert format_btc_amount_as_sats(btc_amount) == "560,000"
```

Comment:
I would have left those tests and used `format_btc_amount_as_sats(btc_amount, enable_digit_formatting=False)` or if we change the default (see other comment by me in the PR), change accordingly, in any case makes sense to test **both**.

Reply:
> in any case makes sense to test **both**.

yes.

Reply:
OK, I'll commit this as well in the next commit.